cmake/FindPTLS.cmake

```diff
@@ -0,0 +1,28 @@
+# - Try to find Picotls
+
+find_path(PICOTLS_INCLUDE_DIR
```

Comment:
Should this be `PTLS_INCLUDE_DIR` here and below (in FPHSA)? Otherwise try to be consisteny and rename PTLS -> PICOTLS everywhere

Reply:
That was a leftover. My wife is suggesting that after dinner programming is a bad idea, especially when the wine was good...